closure/templates/closure_js_template_library.bzl

```diff
@@ -47,6 +48,16 @@ def _impl(ctx):
         for f in dep.closure_js_library.descriptors.to_list():
             args += ["--protoFileDescriptors=%s" % f.path]
             inputs.append(f)
+
+    plugin_classpath = depset(
+        transitive = [m[SoyPluginInfo].classpath for m in ctx.attr.plugin_modules],
```

Comment:
I haven't really used depsets in the past; I wasn't sure if transitive or direct was appropriate. Seems like it probably doesn't matter either way, which also made me unsure about using depset in the first place for this.